ext/reflection/tests/ReflectionProperty_getDefaultValue.phpt

```diff
@@ -0,0 +1,66 @@
+--TEST--
+reflection: ReflectionProperty::getDefaultValue
+--FILE--
+<?php
+
+class TestClass
+{
+    public $foo;
+    public $bar = 'baz';
+
+    public static $static1;
+    public static $static2 = 1234;
+
+    public int $val1;
+    public int $val2 = 1234;
+
+    public ?int $nullable;
+    public ?int $nullable2 = null;
+
+    public $constantAst = 2 * 2;
```

Comment:
This is going to be evaluated during compilation. To get an evaluation at run-time, do something like `public $constantAst = FOO;` and `define('FOO', 42);`.

Reply:
@nikic is that really a thing? Fatals for me

```
Parse error: syntax error, unexpected ''FOO'' (T_CONSTANT_ENCAPSED_STRING), expecting identifier (T_STRING)
```

Reply:
It's a thing: https://3v4l.org/9iRmp

Reply:
added a test, works as expected.